### PR TITLE
More card field separation refactoring

### PR DESCRIPTION
**Pull request by habdelra**
Enforcing contains() only accepts FieldDefs and many renamings

### CODE REVIEW

**Comment by habdelra** [comment]
There are a bunch of cards in the draft realm that aren’t working that need attention still…

File: packages/base/room.gts [hunk 1]
```diff
@@ -251,36 +251,36 @@ class EmbeddedMessageCard extends Component<typeof MessageCard> {
   });
 }
 
-class MessageCard extends FieldDef {
-  @field author = contains(RoomMemberCard);
+class MesageField extends FieldDef {
```
Comment (backspace) [inline packages/base/room.gts]:
```suggestion
class MessageField extends FieldDef {
```

Hopefully `Rename Symbol` makes this easy.

File: packages/host/app/lib/matrix-handlers/index.ts [hunk 2]
```diff
@@ -102,11 +99,11 @@ export async function addRoomEvent(context: Context, event: Event) {
 // work around that, we are performing the assignment of the interior card to
 // the consuming card again which will trigger the consuming card's computeds to
 // pick up the interior card's updated fields. In this case the consuming card is
-// the RoomObjectiveCard and the interior card is the RoomCard.
+// the RoomObjectiveCard and the interior card is the RoomField.
```
Comment (backspace) [inline packages/host/app/lib/matrix-handlers/index.ts]:
```suggestion
// the RoomObjectiveCard and the interior field is the RoomField.
```

I think?

**Review by backspace — CHANGES_REQUESTED** [review]
V minor suggestions

**Review by lukemelia — APPROVED** [review]

**Review by backspace — APPROVED** [review]

**Pull request merged**